instagram/src/components/Post/Post.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import CommentSection from "../CommentSection/CommentSection";
+import PropTypes from "prop-types";
+
+const Post = ({ post }) => {
+  return (
+    <section className="post">
+      <div className="post-header">
+        <img
+          src={post.thumbnailUrl}
+          alt="post.thumbnailUrl"
+          className="user-thumbnail"
+        />
+        <h3>{post.username}</h3>
+      </div>
+      <img src={post.imageUrl} alt={post.imageUrl} className="post-image" />
+      <i className="icon-post far fa-heart" />
+      <i className="icon-post far fa-comment" />
+      <h3>{post.likes} likes</h3>
+      <CommentSection
+        comments={post.comments}
+        timestamp={post.timestamp}
+        className="comment-section"
```

Comment:
adding a className onto a component like this will be passing it down as `props` instead of applying an actual className to it